optimum/commands/export/tflite.py

```diff
@@ -189,7 +189,7 @@ def parse_args_tflite(parser: "ArgumentParser"):
         default=None,
         help=(
             "The name of the column in the dataset containing the main data to preprocess. "
-            "Only for sequence-classification and token-classification. "
+            "Only for text-classification and token-classification. "
```

Comment:
Why replacing "sequence" by "text" here while the task `sequence-classification` is still documented?

Reply:
Fixed in c8b9abb03b84e91c892b316538148d1c899c102e